public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+            $scope.btn__strict = {"box-shadow":"inset 0 0 8px #666"};
+        }
+        else {
+            strict = false;
+            $scope.btn__strict = {"box-shadow":"inset 0 0 1px #666"}
+        }
+
+    };
+
+    startRound = function(round) {
+        $scope.screen = round;
+        var color = colorSequence[count];
+        playSound(color.code);
+        // Light up the color by changing its CSS background-color value
+        $scope[colorCode[color.code]] = color.lightColorObj;
+        turnOffTimer = $timeout(function() {
+            // darken the color by changing back its CSS background-color value
+            $scope[colorCode[color.code]] = color.darkColorObj;
+            recursiveTimer = $timeout(function(){
+                count++;
+                if(count < round)
+                    startRound(round);
+            }, speed/2);
+        }, speed);
+    };
+
+     $scope.passUserInput = function(n) {
+         if(on === true && lock === false) {
+             cancelAllTimers();
+             turnOffAllLights();
+             var result = AppService.addUserInput(n, round);
+             switch(result) {
+                 // user input is incorrect
+                 case 0:
+                    wrongInput();
+                    break;
+                // correct input and round not yet completed
+                 case 1:
+                    break;
+                 // correct input and round is completed
+                 case 2:
+                    round++;
+                    resetRound(true);
+                    break;
+                  // correct input and sequence is completed
+                  case 3:
+                    win.play();
+                    toggleLights(0, 5);
+                    break;
+             }
+         }
+    };
+
+    $scope.lightUp = function(n) {
+        if(on === true && lock === false) {
+            $scope[colorCode[n]] = AppService.getLightColorObj(n);
+            playSound(n);
+        }
+    };
+
+    turnOnAllLights = function() {
+        $scope[colorCode[1]] = AppService.getLightColorObj(1);
+        $scope[colorCode[2]] = AppService.getLightColorObj(2);
+        $scope[colorCode[3]] = AppService.getLightColorObj(3);
+        $scope[colorCode[4]] = AppService.getLightColorObj(4);
+    };
+
+    turnOffAllLights = function() {
+        $scope[colorCode[1]] = AppService.getDarkColorObj(1);
+        $scope[colorCode[2]] = AppService.getDarkColorObj(2);
+        $scope[colorCode[3]] = AppService.getDarkColorObj(3);
+        $scope[colorCode[4]] = AppService.getDarkColorObj(4);
+    };
+
+    // resets everything and generates a new sequence
+    reset = function(restart) {
+        AppService.resetSequence();
+        AppService.generateSequence(AppService.getSequenceLength());
+        // initialize sequence of color objects
+        colorSequence = AppService.getColorSequence();
+        resetToRound1(restart);
+    };
+
+    // resets back to round 1
```

Comment:
Don't tie your function names to numerical positions.  Also would make more sense to move round = 1 into the reset method, and change resetToRound1 to something like resetGameDisplay.